### ISSUE TITLE
Slim version without internal config files 

### ISSUE THREAD

**Issue by stokito** [j256/simplemagic#63]
I see that the lib contains some DB files with magics but I need a small version of dependency which will use only system magic file. Is it possible?

**Comment by j256** [comment]
RTFM.  http://256stuff.com/sources/simplemagic/docs/magic-entries

**Comment by j256** [comment]
Yeah sorry.  I didn't really address this ticket.  I will consider creating a -code extension or something.
